### PR TITLE
Gitops humio

### PR DESCRIPTION
**Pull request by huang-cn**
- Humio Gitops deploy
- Fluentbit Gitops deploy
- Current version still require manual steps to get ingest token, 2 steps to deploy both Humio + Fluentbit

### CODE REVIEW

File: gitops-charts/humio-helm-charts/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# Changelog
```
Comment (gyliu513) [inline gitops-charts/humio-helm-charts/CHANGELOG.md]: what does changelog means here? Does the helm chart from open source?

Comment (huang-cn) [reply gitops-charts/humio-helm-charts/CHANGELOG.md]: the chart files are from open source, the changelog wasn't mine.

**Comment by huang-cn** [comment]
new commit to get both Humio and Fluentbit deployed together, also created automation to get the ingest token, so no UI manaul steps required any more.

File: gitops-charts/humio-helm-charts/CHANGELOG.md (hunk 1 above)
Comment (huang-cn) [reply gitops-charts/humio-helm-charts/CHANGELOG.md]: I've removed the changelog file to avoid confusion.

**Comment by huang-cn** [comment]
Updated few commites to:
- update scc with route
- fix job volumn mount
- fix yaml format

**Pull request merged**